include/rmm/detail/aligned.hpp

```diff
@@ -58,7 +58,21 @@ constexpr std::size_t align_up(std::size_t v, std::size_t align_bytes) noexcept
 }
 
 /**
- * @brief Align down to a power of 2
+ * @brief Check whether `v` is aligned to a multiple of `align_bytes`
+ *
+ * @param v The value to check
+ * @param align_bytes alignment amount, in bytes, must be a power of 2
+ * @return true
+ * @return false
+ */
+constexpr bool is_aligned(std::size_t v, std::size_t align_bytes) noexcept
+{
+  assert(is_supported_alignment(align_bytes));
+  return v & ~(align_bytes - 1);
```

Comment:
Shouldn't this be `!(v & (align_bytes - 1))`?

Reply:
Maybe, but I realized my implementation was missing the point. I fixed it, so this function is gone.